### PR TITLE
BI-9948 Correct retry and error topic naming in configuration

### PR DESCRIPTION
**Pull request by phudson-ch**
* Correct retry and error topic naming in configuration
* Improve logging output by adding attempt to response handler log output

Relates to: [BI-9948 Fix issues with Kafka resilience in Item Handler](https://companieshouse.atlassian.net/browse/BI-9947)

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -18,13 +18,13 @@ application-namespace=item-handler
 kafka.topics.email-send=email-send
 kafka.topics.order-received=order-received
 kafka.topics.order-received_group=${application-namespace}-${kafka.topics.order-received}
-kafka.topics.order-received-notification-retry=order-received-notification-retry
+kafka.topics.order-received-notification-retry=order-received-retry
```
Comment (dlgroves-ch) [inline src/main/resources/application.properties:21]: Please rename this property `kafka.topics.order-received-retry`

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -18,13 +18,13 @@ application-namespace=item-handler
...
 kafka.topics.order-received-notification-retry-group=${application-namespace}-${kafka.topics.order-received-notification-retry}
-kafka.topics.order-received-notification-error=order-received-notification-error
+kafka.topics.order-received-notification-error=order-received-error
```
Comment (dlgroves-ch) [inline src/main/resources/application.properties:23]: Please rename this property `kafka.topics.order-received-error`

File: src/test/resources/application.properties [hunk 3]
```diff
@@ -16,9 +16,9 @@ application-namespace=item-handler
 kafka.topics.email-send=email-send
 kafka.topics.order-received=order-received
 kafka.topics.order-received_group=${application-namespace}-${kafka.topics.order-received}
-kafka.topics.order-received-notification-retry=order-received-notification-retry
+kafka.topics.order-received-notification-retry=order-received-retry
```
Comment (dlgroves-ch) [inline src/test/resources/application.properties:19]: Please rename this property `kafka.topics.order-received-retry`

File: src/test/resources/application.properties [hunk 4]
```diff
@@ -16,9 +16,9 @@ application-namespace=item-handler
...
 kafka.topics.order-received-notification-retry-group=${application-namespace}-${kafka.topics.order-received-notification-retry}
-kafka.topics.order-received-notification-error=order-received-notification-error
+kafka.topics.order-received-notification-error=order-received-error
```
Comment (dlgroves-ch) [inline src/test/resources/application.properties:21]: `Please rename this property kafka.topics.order-received-error`

**Review by dlgroves-ch — CHANGES_REQUESTED** [review]
Please rename `kafka.topics.order-received-notification-*` properties and all usages to `kafka.topics.order-received-*`.

File: src/main/resources/application.properties [hunk 5]
```diff
@@ -18,13 +18,13 @@ application-namespace=item-handler
...
 kafka.topics.order-received-notification-error-group=${application-namespace}-${kafka.topics.order-received-notification-error}
 kafka.topics.chd-item-ordered=chd-item-ordered
 
 # Order process response handler
-response.handler.maximumRetryAttempts = 5
+response.handler.maximumRetryAttempts = 1
```
Comment (cgregoryCH) [inline src/main/resources/application.properties:28]: Should this value be 1 or 5?

Comment (phudson-ch) [reply src/main/resources/application.properties]: Should be 5 - fixed

**Comment by phudson-ch** [comment]
No longer required - change made as part of another PR

**Pull request closed without merging**